test/e2e/app-dir/app-static/app/variable-revalidate/post-method-cached/page.js

```diff
@@ -93,9 +75,8 @@ export default async function Page() {
       <p id="page-data">{data}</p>
       <p id="data-body1">{dataWithBody1}</p>
       <p id="data-body2">{dataWithBody2}</p>
-      <p id="data-body3">{dataWithBody3}</p>
+      <p id="data-body4">{dataWithBody3}</p>
```

Comment:
typo?

Reply:
Yeah good catch, updated in https://github.com/vercel/next.js/pull/46645